src/components/Chip/index.js

```diff
@@ -29,6 +29,7 @@ export default function Chip(props) {
                 <ButtonIcon
                     className="rainbow-chip_button-container"
                     icon={<CloseIcon />}
+                    tabIndex={-1}
```

Comment:
Remove this since this make the chip close button not focusable and it must be focusable.
Look at:
https://www.lightningdesignsystem.com/components/lookups/#site-main-content
in order to view the right behavior